components/public-api/gitpod/v1/workspace.proto

```diff
@@ -38,6 +43,28 @@ message WatchWorkspaceStatusResponse {
   WorkspaceStatus status = 2;
 }
 
+message ListWorkspacesRequest {
+  // pagination contains the pagination options for listing workspaces
+  PaginationRequest pagination = 1;
+
+  // organization_id is the ID of the organization that contains the workspaces
+  string organization_id = 2;
+
+  // pinned indicates whether to list only pinned workspaces
+  bool pinned = 3;
```

Comment:
Do we have a use-case for this in our clients or in the Dashboard redesigns?

Reply:
We only define proto based on Dashboard usage now, so you can check old `list-workspaces-query.ts` file, that we need `pinned` param

Reply:
I see, thank you!